src/components/ProductPage/BodySection/BodySection.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { SectionProps } from '../HomepageContent';
+import { BodySectionDescription } from './BodySectionDescription';
+import { FeatureCard } from './Card/FeatureCard';
+
+const cardTypes = {
+  feature: FeatureCard,
+};
+
+const betaPillStyle = {
+  fontSize: '14px',
```

Comment:
I think for these we should use CSS variables where we can, like:

~~~js
{
  fontSize: 'var(--fs-p3)', // should be 14px
}
~~~

Line height of `1.25rem` can be done with the `leading-normal` class from tailwind.